tests/UI/pages/BO/orders/view.js

```diff
@@ -83,6 +83,26 @@ class Order extends BOBasePage {
     return this.getTextContent(page, `${this.orderStatusesSelect} option[selected='selected']`, false);
   }
 
+  /**
+   * Is status exist
+   * @param page
+   * @param statusName
+   * @returns {Promise<boolean>}
+   */
+  async isStatusExist(page, statusName) {
```

Comment:
I'm not a perfect english speaker but something feels off with this name. @matks do you have a suggestion ?

Reply:
```suggestion
  async doesStatusExist(page, statusName) {
```
Indeed 😉 `is` must be followed by a noun like `isName` `isInteger` `isValid` ; but the verb here is `Exist` so `does` must be used

Reply:
Done thnx :+1: